There does not seem to be any issue with console-bridge >= 1.0
Also, all supported distributions have console-bridge >= 0.3

--- CMakeLists.txt~	2019-01-29 20:47:47.000000000 +0100
+++ CMakeLists.txt	2022-05-18 12:22:50.170825057 +0200
@@ -41,7 +41,7 @@
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
-find_package(console_bridge 0.3 REQUIRED)
+find_package(console_bridge REQUIRED)
 include_directories(SYSTEM ${console_bridge_INCLUDE_DIRS})
 link_directories(${console_bridge_LIBRARY_DIRS})
 
